### ISSUE TITLE
Get Performance Tests Working

### ISSUE THREAD

**Issue by miniksa** [microsoft/Terminal#779]
I've added a performance mode to some of the `conhost.feature.tests.dll` tests and managed to ensure that they do work with the WinPerf infrastructure.

Now I need to finish up and get this running in an automated fashion. I'm pretty sure that WinPerf and the Windows Performance backend services are going to remain private, but that won't necessarily stop us from having an internal build that can orchestrate these and submit bugs that we can then pull back out to GitHub.

**Comment by zadjii-msft** [comment]
@miniksa This is different than the PGO thing, right? This is more about the automated perf regression testing, yea?

**Comment by miniksa** [comment]
@zadjii-msft yes you're right. It's about an automated lab for regression tests
